jira.yaml.example

```diff
@@ -1,10 +1,29 @@
 ---
-  url: "https://company.atlassian.net"
-  rsa_cert_file: "/home/vagrant/jira.pem"
-  auth_method: "oauth"
-  oauth_token: ""
-  oauth_secret: ""
-  consumer_key: ""
-  poll_interval: 30
-  project: "MY_PROJECT"
-  verify: True
+url: "https://company.atlassian.net"
+rsa_cert_file: "/home/vagrant/jira.pem"
+auth_method: "oauth"
+oauth_token: ""
+oauth_secret: ""
+consumer_key: ""
+poll_interval: 30
+project: "MY_PROJECT"
+verify: True
+default_profile: "dev"
+profiles:
+  - name: "dev"
```

Comment:
Since your `config.schema.yaml` file specifies that `profiles` should be a dictionary/object, make sure you update the `profiles` attribute in this example config to be a dictionary/object and _not_ an array.